webui/react/src/components/PageHeaderFoldable.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Dropdown, Menu } from 'antd';
+import React, { CSSProperties, useState } from 'react';
+
+import Icon from 'components/Icon';
+
+import css from './PageHeaderFoldable.module.scss';
```

Comment:
hover states needs to be revisited, @hkang1 have you already ideas about this?

Reply:
I'll need to check out this branch and try it out to get back to you on this point.